### PR TITLE
Blog post about me and my new role

### PR DESCRIPTION
**Pull request by weaverbel**
I have created this to chime with the announcement that I am soon coming on board. Comments welcome !

### CODE REVIEW

File: _posts/2017/06/2017-06-06-bw_cdl.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+especially foreign/art house ones, and grow herbs and vegetables in the garden. 
+
+I also love teaching - whether it be Software Carpentry workshops, Library Carpentry courses, or 
+training new instructors. 
+
+I dive right into the new job on 19 June, when I will be running an instructor training event with Jonah Duckles 
+at Macquarie University in Sydney. It will be great to see the instructor pool in Sydney increase.
+
+I have been a fan of Software Carpentry since first hearing about it in 2014, when I organised the first ever Software 
+Carpentry bootcamp in Brisbane in July.
```
Comment (jduckles) [inline _posts/2017/06/2017-06-06-bw_cdl.md]: "...July of that year."

**Review by jduckles — APPROVED** [review]
Looks good. Thanks for writing this up!

File: _posts/2017/06/2017-06-06-bw_cdl.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+I have been a fan of Software Carpentry since first hearing about it in 2014, when I organised the first ever Software 
+Carpentry bootcamp in Brisbane in July of that year.
+
+I certified as a Software Carpentry instructor in 2015 (and as a Data Carpentry instructor the same year), and 
+taught at two workshops. In 2016, I taught at eight workshops out of a total of 14 statewide. During 2016, I and other 
+Queensland instructors took Software Carpentry to five cities in Queensland - Brisbane, Townsville, Toowoomba, 
+Gold Coast and Rockhampton - a huge improvement on 2015, when we taught three workshops in Brisbane only. 
+
+I organised Software Carpentry instructor training in Brisbane in 2016, and certified as an instructor trainer myself in 
+late 2016. I have since helped train an online cohort and a librarian cohort face to face in Portland, OR (with the wonderful
```
Comment (rgaiacs) [inline _posts/2017/06/2017-06-06-bw_cdl.md]: Replace "OR" with "Oregon". :-)

**Comment by weaverbel** [comment]
Thanks @rgaiacs - I amended - also fixed a typo while I was in there! I think it's good to go now.

**Comment by weaverbel** [comment]
Blinking GitHub! I wanted to create my Library Carpentry sprint write up as a separate PR but it's got bundled in here with my other post. Happy to have it decoupled. Would do it myself but not sure how to remove that file from the PR. SORRY.

**Comment by tracykteal** [comment]
Can you revert your last commit? Then the LC post could be put in as a separate one.

**Comment by rgaiacs** [comment]
> Blinking GitHub! I wanted to create my Library Carpentry sprint write up as a separate PR but it's got bundled in here with my other post. Happy to have it decoupled. Would do it myself but not sure how to remove that file from the PR. SORRY.

This isn't a GitHub issue but a Git related issue. You should use "feature branch" instead of gh-pages. If you are using the GitHub web interface I don't know how you can create a branch so I can't help you. If you are using the command line you can create your "feature branch" with

~~~
$ git checkout 503d10c
$ git checkout -b library-carpentry-sprint
$ git cherry-pick c9b88ed
$ git push origin library-carpentry-sprint
$ git checkout gh-pages
$ git revert --hard HEAD~
$ git push -f origin gh-pages
~~~

The explanation for each command before is

1. Go to the commit before this pull request
2. Create a new (feature) branch
3. Copy your "library carpentry sprint" commit from the gh-pages branch to the branch created on the previous step
4. Push your new branch to GitHub
5. Go back to the gh-pages branch (the one used on this pull request)
6. Remove your "library carpentry sprint" commit  from the gh-pages
7. Push your changes on the gh-pages branch to GitHub

GitHub will automatic update this pull request. You will need to create a new pull request using the new branch, library-carpentry-sprint.

**Comment by weaverbel** [comment]
I am now at work so no terminal access, so i think I will close this PR, and start again.

**Pull request closed without merging**